### PR TITLE
Adding check and bumping version numbers

### PR DESCRIPTION
**Pull request by callixte**
I added a minimal check so that messages without an 'ids' property are rejected. It doesn't satisfy me completely, but will be enough for now.
I also updated the version of node required as tests fail with older version.
And bumped to 0.2.0 as the API has changed.

### CODE REVIEW

**Comment by ovaussy-virtuoz** [comment]
as the travis build passed SHIPIT :D

File: lib/smartrouter.js [hunk 1]
```diff
@@ -323,6 +323,12 @@ module.exports = new JS.Class(EventEmitter, {
         return callback(err, false);
       }
       // here we can add additional checks.
+      if (!message.ids) {
+        var error = util.format("Received an ill-formed message, missing ids. (%s)", message);
```
Comment (sbellone) [inline lib/smartrouter.js:327]: You have to stringify the message, here you will see [object Object], not very useful to debug.

**Comment by sbellone** [comment]
Travis tests ok, but manual testing failed! Unship! :D

File: lib/smartrouter.js (hunk 1 above)
Comment (callixte) [inline lib/smartrouter.js:327]: Yep you are right. I am getting rusted and forgot the good reflex.

**Comment by callixte** [comment]
What manual tests?
I have added the stringify and pushed to npm repo.

**Comment by sbellone** [comment]
Never mind. Thanks, so we can update the version in our virtuozlabs repo.

**Pull request merged**